src/credssp/sspi_cred_ssp/tls_connection.rs

```diff
@@ -117,20 +153,21 @@ impl TlsConnection {
     pub fn stream_sizes(&self) -> Result<StreamSizes> {
         match self {
             TlsConnection::Rustls(tls_connection) => {
-                let connection_cipher = tls_connection
-                    .negotiated_cipher_suite()
-                    .ok_or_else(|| Error::new(ErrorKind::InternalError, "Connection cipher is not negotiated"))?;
-
-                let bulk_cipher = match connection_cipher {
-                    rustls::SupportedCipherSuite::Tls12(cipher_suite) => &cipher_suite.common.bulk,
-                    rustls::SupportedCipherSuite::Tls13(cipher_suite) => &cipher_suite.common.bulk,
-                };
-                let block_size = match bulk_cipher {
-                    rustls::BulkAlgorithm::Aes128Gcm => AES_BLOCK_SIZE,
-                    rustls::BulkAlgorithm::Aes256Gcm => AES_BLOCK_SIZE,
-                    // ChaCha20 is a stream cipher
-                    rustls::BulkAlgorithm::Chacha20Poly1305 => 0,
-                };
+                // let connection_cipher = tls_connection
+                //     .negotiated_cipher_suite()
+                //     .ok_or_else(|| Error::new(ErrorKind::InternalError, "Connection cipher is not negotiated"))?;
+                //
+                // let bulk_cipher = match connection_cipher {
+                //     rustls::SupportedCipherSuite::Tls12(cipher_suite) => &cipher_suite.common.bulk,
+                //     rustls::SupportedCipherSuite::Tls13(cipher_suite) => &cipher_suite.common.bulk,
+                // };
+                // let block_size = match bulk_cipher {
+                //     rustls::BulkAlgorithm::Aes128Gcm => AES_BLOCK_SIZE,
+                //     rustls::BulkAlgorithm::Aes256Gcm => AES_BLOCK_SIZE,
+                //     // ChaCha20 is a stream cipher
+                //     rustls::BulkAlgorithm::Chacha20Poly1305 => 0,
+                // };
+                let block_size = AES_BLOCK_SIZE;
```

Comment:
issue: This commit seems to include many changes unrelated to the Mutex addition